examplebroker/broker.go

```diff
@@ -286,11 +286,6 @@ func (b *Broker) NewSession(ctx context.Context, username, lang, mode string) (s
 		return "", "", fmt.Errorf("user %q does not exist", username)
 	}
 
-	if info.sessionMode == auth.SessionModePasswd {
-		info.neededAuthSteps++
-		info.pwdChange = mustReset
-	}
-
```

Comment:
>  examplebroker: Fix support for integration tests users in passwd mode 

It would be nice if you could explain in the commit message what was broken and why this change fixes it.

Reply:
done